extendr-api/src/wrapper/matrix.rs

```diff
@@ -499,7 +517,7 @@ mod tests {
             vec![15.0, 16.0, 22.0, 3.0, 9.0].try_into().unwrap(),
         ];
         let (n_x, n_y) = (5, 5);
-        let matrix = RMatrix::new_matrix(n_x, n_y, |r, c| res[c][r]);
+        let _matrix = RMatrix::new_matrix(n_x, n_y, |r, c| res[c][r]);
```

Comment:
See, I thought we should check here.. but check what lol. Not my addition.

Reply:
i can revert if needed! I used `_` to avoid the warning

Reply:
No, no.. I just wish someone would make a decision about this test.